src/main/kotlin/no/nav/syfo/kartleggingssporsmal/infrastructure/kafka/consumer/kartleggingssporsmalsvar/KartleggingssporsmalSvarConsumerTask.kt

```diff
@@ -0,0 +1,41 @@
+package no.nav.syfo.kartleggingssporsmal.infrastructure.kafka.consumer.kartleggingssporsmalsvar
+
+import no.nav.syfo.ApplicationState
+import no.nav.syfo.shared.infrastructure.kafka.KafkaEnvironment
+import no.nav.syfo.shared.infrastructure.kafka.kafkaAivenConsumerConfig
+import no.nav.syfo.shared.infrastructure.kafka.launchKafkaConsumer
+import no.nav.syfo.shared.util.configuredJacksonMapper
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.clients.consumer.OffsetResetStrategy
+import org.apache.kafka.common.serialization.Deserializer
+
+const val KARTLEGGINGSSPORSMAL_SVAR_TOPIC =
+    "team-esyfo.kartleggingssporsmal-svar"
+
+fun launchKartleggingssporsmalSvarConsumer(
+    applicationState: ApplicationState,
+    kafkaEnvironment: KafkaEnvironment,
+) {
+    val consumerProperties = kafkaAivenConsumerConfig<KafkaKartleggingssporsmalSvarDTODeserializer>(
+        kafkaEnvironment = kafkaEnvironment,
+        offsetResetStrategy = OffsetResetStrategy.LATEST,
```

Comment:
Litt avhengig av timing for prodsetting, så kan det hende denne skal være `EARLIEST`.

Reply:
Kanskje best å operere med EARLIEST? (og så kan man sjekke om svarId finnes fra før og skippe records som allerede er mottatt)